### PR TITLE
Competitions map

### PR DESCRIPTION
**Pull request by AnthonyCamper**

### CODE REVIEW

**Comment by AnthonyCamper** [comment]
Looks good to me

File: public/json/clubs.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {"school":"School Name",
+        "address":"123 School Address Road\n",
+        "club":"Club Name",
+        "city":"City, State Zip",
+        "phone":"(555) 555-5555",
+        "website":"https://www.schoolwebsite.club/",
+        "latitude":0,
+        "longitude":-0,
+        "isredvsblue":true,
+        "isctf":false},
```
Comment (Cyb3r-Jak3) [inline public/json/clubs.json]: Can you remove this fake school.

**Review by Cyb3r-Jak3 — COMMENTED** [review]
I noticed that there is a random pop-up on the page. This should be removed

original

**Review by Cyb3r-Jak3 — CHANGES_REQUESTED** [review]
Please look at comments

**Comment by AnthonyCamper** [comment]
> I noticed that there is a random pop-up on the page. This should be removed
> 
> original

Removed lines

**Review by Cyb3r-Jak3 — APPROVED** [review]
LGTM! Nice work

**Pull request merged**